src/table/components/TableBodyWrapper.jsx

```diff
@@ -63,14 +44,31 @@ function TableBodyWrapper({
     addSelectionListeners({ api: selectionsAPI, selectionDispatch, setShouldRefocus, keyboard, tableWrapperRef });
   }, []);
 
+  const tableBodyStyle = {
+    '& td, th': {
+      fontSize: bodyStyle.fontSize,
```

Comment:
why is color not here anymore and passed on styling instead?

Reply:
to remove the !important
<img width="821" alt="Screenshot 2022-02-11 at 10 59 42" src="https://user-images.githubusercontent.com/4471489/153571786-13480240-5725-4560-8cea-e76187b9d109.png">

Reply:
ahh great then :)